config/packages/doctrine.yaml

```diff
@@ -27,6 +27,10 @@ when@prod:
             result_cache_driver:
                 type: pool
                 pool: doctrine.result_cache_pool
+            controller_resolver:
+                enabled: true
+                auto_mapping: true
```

Comment:
I would actually disable auto_mapping here. For cases where the URL contains the id, it is not necessary. And for the case where the URL contains a slug, you are using an explicit mapping because of the different name. So I don't think auto_mapping is used in this project.
Btw, in my experience, this auto_mapping behavior can have surprising results (especially when using optional arguments), which is precisely why I advocated to make it configurable (in the DoctrineParamConverter of SFEB, there was no way to turn it off and so explicit `@ParamConverter` annotation was necessary to override it in case of buggy behavior introduced by it)

Reply:
Thanks. I've just removed it. I read the docs about this and I thought it was necessary.

My local app didn't work, I added this and it worked ... but now I remove it and the app still works. So I guess it's not needed.